.github/workflows/main.yml

```diff
@@ -163,14 +155,14 @@ jobs:
             -CIUserToken $env:SIGNPATH_USER_TOKEN `
             -OrganizationId $env:SIGNPATH_ORG_ID `
             -ProjectSlug Endless_Apps `
-            -SigningPolicySlug Endless_Apps_Test_Signing_for_Microsoft_Store `
+            -SigningPolicySlug Endless_Apps_Signing_for_Microsoft_Store `
             -ArtifactConfigurationSlug Endless_Key_APPX_Package_for_Microsoft_Store `
             -InputArtifactPath endlesskeyapp.unsigned.appx `
             -OutputArtifactPath endlesskeyapp.appx `
             -WaitForCompletion
 
-      - name: Upload signed APPX
-        if: steps.sign_for_release.outcome == 'success' || steps.sign_for_testing.outcome == 'success'
+      - name: Upload signed APPX for release
```

Comment:
Here as well, we don't need to sign anything during a release, so this step is not needed anymore.

Reply:
ok